### PR TITLE
docs: add dev docs for aa-accounts/light-account

### PR DESCRIPTION
**Pull request by avasisht23**
https://github.com/OMGWINNING/aa-sdk-private/assets/43521356/49276fb6-8274-4aea-b79c-9deca8e8d3ed

### CODE REVIEW

**Comment by avasisht23** [comment]
Current dependencies on/for this PR:
* main
  * **PR #19** <a href="https://app.graphite.dev/github/pr/OMGWINNING/aa-sdk-private/19" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/OMGWINNING/aa-sdk-private/19?utm_source=stack-comment).

File: site/packages/aa-accounts/light-account/encodeTransferOwnership.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: LightSmartContractAccount
```
Comment (moldy530) [inline site/packages/aa-accounts/light-account/encodeTransferOwnership.md:6]: Nit: let's make the title the same as the method or at least LightSmartContractAccount • {method name}

File: site/packages/aa-accounts/light-account/introduction.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      content: Overview of the LightSmartContractAccount class in aa-accounts
+---
+
+# Light Account
+
+`LightSmartContractAccount` is a simple, secure, and cost-effective smart account implementation which extends `SimpleSmartContractAccount` as an implementation of `BaseSmartContractAccount`. It supports features such as owner transfers, [EIP-1271](https://eips.ethereum.org/EIPS/eip-1271) message signing, and batched transactions. We recommend using Light Account for most use cases.
+
+Notable differences between `LightSmartContrctAccount` and `SimpleSmartContractAccount` are implementations for:
+
+1.  `signMessageWith6492` -- supports message signatures for deployed smart contract accounts, as well as undeployed accounts (counterfactual addresses) using [EIP-6492](https://eips.ethereum.org/EIPS/eip-6492).
```
Comment (moldy530) [inline site/packages/aa-accounts/light-account/introduction.md:21]: For this and the methods below, let's have them link to the docs for the method too

File: site/smart-accounts/transferring-ownership.md [hunk 3]
```diff
@@ -90,4 +90,4 @@ const hash = LightSmartContractAccount.transferOwnership(provider, newOwner); //
 
 :::
 
-see the [`LightSmartContractAccount`](/packages/aa-accounts/light-account) docs for more details about this class.
+see the [`LightSmartContractAccount`](/packages/aa-accounts/light-account/introduction.html) docs for more details about this class.
```
Comment (moldy530) [inline site/smart-accounts/transferring-ownership.md:93]:
```suggestion
see the [`LightSmartContractAccount`](/packages/aa-accounts/light-account/introduction) docs for more details about this class.
```

**Review by moldy530 — APPROVED** [review]

**Pull request merged**